cli/remotebazel/remotebazel.go

```diff
@@ -551,6 +572,7 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		Arch:               reqArch,
 		ContainerImage:     *containerImage,
 		Env:                envVars,
+		InvocationId:       invocationID,
```

Comment:
nit: since the response returns an invocation ID can we use that instead? cancelling the invocation before we get a response back might not do anything anyway, right?

Reply:
Good idea - changed